### PR TITLE
Use baseUrl for github-pages deployment

### PR DESCRIPTION
**Pull request by marcin-krystianc**
Achieves the same effect as https://v2.nuxt.com/docs/configuration-glossary/configuration-router/#base, but works with Nuxt 3.

### CODE REVIEW

**Comment by baimamboukar** [comment]
Hello @marcin-krystianc ! This configuration seems to be working...
What is left please ?

**Comment by marcin-krystianc** [comment]
> Hello @marcin-krystianc ! This configuration seems to be working...

I don't think it works (although it should). I don't know yet what is going on.

File: _dashboard/DotNetPerfMonitor/nuxt.config.ts [hunk 1]
```diff
@@ -1,13 +1,23 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
   devtools: { enabled: true },
+  target: 'static',
+  app: {
+	baseURL: '/DotNetPerfMonitor/',
```
Comment (marcin-krystianc) [inline _dashboard/DotNetPerfMonitor/nuxt.config.ts:6]: Fixes problem with URL prefixes when deployed to GitHub Pages

File: _dashboard/DotNetPerfMonitor/nuxt.config.ts [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+  
   colorMode: {
     preference: "dark",
   },
   modules: ["@nuxthq/ui"],
 
   css: ["@/assets/css/global.css"],
+  
+  experimental: {
+	payloadExtraction: "false",
```
Comment (marcin-krystianc) [inline _dashboard/DotNetPerfMonitor/nuxt.config.ts:21]: I'm not sure if disabling the experimental payload extraction is necessary, but I did it anyway to get rid of a warning.

**Review by marcin-krystianc — COMMENTED** [review]
@baimamboukar 
Finally, I've found how to make the routing work when the page is deployed to GitHub pages.

File: _dashboard/DotNetPerfMonitor/nuxt.config.ts (hunk 1 above)
Comment (baimamboukar) [reply _dashboard/DotNetPerfMonitor/nuxt.config.ts:6]: This is cool! I tried to add this configuration.... But I directly put target: 'static', and baseURL: '/DotNetPerfMonitor'...  Did not know I had to wrap baseURL key around app object!

**Review by baimamboukar — COMMENTED** [review]
Looks pretty clean! Let's see how it goes....

**Review by baimamboukar — APPROVED** [review]

**Pull request merged**